src/dagmc.cpp

```diff
@@ -395,7 +417,10 @@ void DAGUniverse::to_hdf5(hid_t universes_group) const
 
 bool DAGUniverse::uses_uwuw() const
 {
-  return !uwuw_->material_library.empty();
+  if (uwuw_disabled)
```

Comment:
I think relying purely on information about the UWUW shared pointer will make this more straightforward:

```cpp
return uwuw_ && !uwuw_->material_library.empty();

Reply:
done